internal/provider/private_endpoint_connection_resource_test.go

```diff
@@ -38,17 +38,18 @@ func TestAccDedicatedPrivateEndpointConnectionResource(t *testing.T) {
 }
 
 func TestAccServerlessPrivateEndpointConnectionResource(t *testing.T) {
-	t.Skip("Skipping until we can either integrate the AWS provider " +
-		"or import a permanent test fixture.")
+	// This test relies on a pre-created private endpoint created via the AWS
```

Comment:
This feels a little fragile to me.  Do we have anything protecting it from being removed?  Many teams won't have the perms to fix it if it breaks.

Reply:
I can't think of a way to make it "unfragile", except actually creating an AWS service account, an API key for the account, adding AWS specific logic to create and delete the private endpoint connection on each test run, running the tests with the AWS API key. We'd also need to figure out a way to store AWS API Key so all engineers have access? Maybe vault, but then the TF tests would have a vault dep. I don't think it's a viable approach atm, too much hassle.

I know it's janky but I think the approach in the PR could get us working acceptance tests for a year. It will break eventually and it be annoying to fix but IMO it's worth having the coverage.

Reply:
Hmm actually I'm going to leave this test as skipped since the endpointId only works for staging. Any acceptance test ran against any other env will fail.

Reply:
Is there a way to make it conditional on the env?  Or maybe we can add instructions for running the test in staging?